c/include/libsbp/sbp.h

```diff
@@ -76,12 +81,20 @@ typedef struct {
   u8 msg_buff[256];
   void* io_context;
   sbp_msg_callbacks_node_t* sbp_msg_callbacks_head;
+  sbp_allmsg_fn_t allmsg_fn;
+  void *allmsg_context;
 } sbp_state_t;
 
 /** \} */
 
+/* registers a callback to be called on a particular msg_type */
 s8 sbp_register_callback(sbp_state_t* s, u16 msg_type, sbp_msg_callback_t cb, void* context,
                          sbp_msg_callbacks_node_t *node);
+/* sets a function to be called on ANY deframed message */
+s8 sbp_set_allmsg_fn(sbp_state_t *s, sbp_allmsg_fn_t fn, void *context);
```

Comment:
How about `sbp_register_allmsg_callback` to match `sbp_register_callback`?

Reply:
I believe I suggested `set` since this is not registered, as in place in a registry of callbacks along with other callbacks. there is only one `allmsg` function to either set or clear

Reply:
I was worried about calling it a "callback".  If I do, then I think it makes sense that sbp_clear_callbacks clears this allmsg callback as well, which is kind of a change in behavior for that part of the API that Ben wasn't too fond of.